views/dataoutputs.erb

```diff
@@ -22,7 +22,7 @@
              [
                ['ElasticSearch + Kibana', 'elasticsearch.png', 'http://docs.fluentd.org/articles/free-alternative-to-splunk-by-fluentd'],
                ['Splunk', 'splunk.jpg', 'http://docs.fluentd.org/v0.12/articles/out_splunk'],
-               [],
+               ['Sumo Logic', 'sumo-logic.png', 'https://github.com/SumoLogic/fluentd-output-sumologic'],
```

Comment:
You shouldn't add direct repository link.
We should create a document for SumoLogic in https://github.com/fluent/fluentd-docs.

Reply:
@cosmo0920 : will file a PR for that soon, thanks for the tip

Reply:
Ah, some of other links use direct repository link.
We can add it....

Reply:
Oh great, if we are good with that short term, I can happily get an official doc later...